server/server/src/billing/google_play_client.rs

```diff
@@ -0,0 +1,68 @@
+use google_androidpublisher3::api::{
+    SubscriptionPurchase, SubscriptionPurchasesAcknowledgeRequest,
+};
+use google_androidpublisher3::hyper::StatusCode;
+use google_androidpublisher3::{AndroidPublisher, Error};
+
+const PACKAGE_NAME: &str = "app.lockbook";
+
+#[derive(Debug)]
+pub enum SimpleGCPError {
+    PurchaseTokenNotFound,
+    Unexpected(String),
+}
+
+impl From<google_androidpublisher3::Error> for SimpleGCPError {
```

Comment:
`google_androidpublisher3::Error` is already imported here. Choose whichever you want (imported or not) throughout the file.